jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/DockerCredentialHelper.java

```diff
@@ -88,10 +106,30 @@ public DockerCredentialHelper(String serverUrl, Path credentialHelper) {
   public Credential retrieve()
       throws IOException, CredentialHelperUnhandledServerUrlException,
           CredentialHelperNotFoundException {
+    boolean isWindows =
+        systemProperties.getProperty("os.name").toLowerCase(Locale.ENGLISH).contains("windows");
+    if (!isWindows || credentialHelper.toString().toLowerCase(Locale.ENGLISH).endsWith(".cmd")) {
+      return retrieve(Arrays.asList(credentialHelper.toString(), "get"));
+    }
+
+    try {
+      return retrieve(Arrays.asList(credentialHelper.toString() + ".cmd", "get"));
```

Comment:
If I recall correctly, .exe files have higher precedence over .cmd files.  It would be worth checking as it would simplify this code.

Reply:
Other file extensions including `.EXE` just don't work, and I decided to keep it that way until we get a new report.

From https://github.com/GoogleContainerTools/jib/issues/2399#issuecomment-612972912:

> Therefore, it may be necessary to also try `.EXE`, `.COM`, `.BAT`, etc. However, we had not received a report until now, so I think I'll just try `.CMD` first and watch.

Reply:
The current PR works, but on second thought, checking `.exe` will give a better error message if Jib can't find the binary. Otherwise, it'll be like "some-helper.exe.cmd not found" instead of "some-helper.exe not found."